concourse/pipelines/linux-image-build.jsonnet

```diff
@@ -230,25 +230,68 @@ local CDSImgBuildJob(image, workflow) = imgbuildjob {
       },
     },
     {
-      task: 'lego-provision-tls-crt',
-      config: lego.legotask {
-        domains: ['rhui.googlecloud.com', 'staging-rhui.googlecloud.com'],
-        acme_server: acme_server,
-        email: acme_email,
+      task: 'get-rhui-tls-key',
+      config: gcp_secret_manager.getsecrettask {
+        secret_name: 'rhui_tls_key',
         project: rhui_project,
-        input: 'gcp-secret-manager',
+
+        // Layer onto the same output as previous task
+        inputs+: gcp_secret_manager.getsecrettask.outputs,
+      },
+    },
+    {
+      task: 'generate-csr',
+      config: {
+        platform: 'linux',
+        image_resource: {
+          type: 'registry-image',
+          source: { repository: 'alpine/openssl' },
+        },
+        run: {
+          path: 'openssl',
+          args: [
+            'req',
+            '-new',
+            '-key=./gcp-secret-manager/rhui_tls_key.pem',
+            '-subj=/CN=rhui.googlecloud.com',
+            '-out=./gcp-secret-manager/thecsr.pem',
```

Comment:
You don't need to declare `gcp-secret-manager` as an input and output?

Reply:
thanks for catching that!